scripts/deploy-community-operator.sh

```diff
@@ -44,10 +44,15 @@ else
 	export CATALOG_NAMESPACE="openshift-marketplace"
 fi
 
+mkdir -p ./temp
+
 # Pre-pull the image for the operator
 docker pull "$COMMUNITY_OPERATOR_IMAGEREPO"/"$COMMUNITY_OPERATOR_BASE":"$COMMUNITY_OPERATOR_IMAGEVERSION"
```

Comment:
The whole block should be in the condition.

Reply:
Fixed.